test/test.js

```diff
@@ -0,0 +1,126 @@
+var assert = require('assert');
+const t = require('../index');
+const {
+    FlightPriceWatcher
+} = require('../index');
+
+// /!\ Go to the botton and add the API KEY before testing /!\
+
+describe('FlightPriceWatcher', () => {
+    describe('#constructor()', function () {
+        it('should construct', function () {
+            let app = new FlightPriceWatcher();
+            assert.ok(app);
+        });
+    });
+    describe('#run()', () => {
+        // Check if the keyboard key is the right color when none is stored
+        // Means getLastPrice returns null
+        it('Can get price when none is stored', function () {
+            let app = new FlightPriceWatcher();
+            assert.ok(app.setLastPrice(null));
+            assert.equal(app.getLastPrice(), null);
+            // Check if the date format is correct
+            assert.ok(app.formatDate());
+          })
+        it('check the color of the key when none is stored', async function () {
+            const config = getConfig();
+            let app = await buildApp(config);
+            return app.run().then((signal) => {
+                assert.ok(signal); 
+                // Has to be green
+                assert.equal(signal.points[0][0].color, '#088A08');
+            }).catch((error) => {
+                assert.fail(error)
+            });
+        });
+    });
+    describe('FlightPriceWatcher', () => {
+        // Check if the function getPrice give the correct current price
+        it('Can get the current price', async function () {
+            const config = getConfig();
+            let app = await buildApp(config);
+            return app.getPrice(config.authorization.apiKey).then((new_price) => {
+                console.log('<<<<<get current price>>>>', new_price);
+                assert.notEqual(new_price, null);
+            }).catch((error) => {
+                assert.fail(error);
+            })
+        })
```

Comment:
Wrong test. You are testing something that is not working. You don't provide any API key. 
If you want to test the function getPrice for real you can add a file test/credential.json where you add your API_KEY.
Careful this file should never be comitted ( add it to the .gitignore)